sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/xml/TestCase.java

```diff
@@ -23,234 +23,85 @@
 import com.thoughtworks.xstream.annotations.XStreamAsAttribute;
 import com.thoughtworks.xstream.annotations.XStreamOmitField;
 import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang.builder.ToStringBuilder;
+import org.sonar.api.internal.google.common.annotations.VisibleForTesting;
 
-/**
- * The Class TestCase.
- */
 @XStreamAlias("testcase")
 public final class TestCase {
 
-  /**
-   * The Constant STATUS_ERROR.
-   */
-  public static final String STATUS_ERROR = "error";
-
-  /**
-   * The Constant STATUS_FAILURE.
-   */
-  public static final String STATUS_FAILURE = "failure";
-
-  /**
-   * The Constant STATUS_OK.
-   */
-  public static final String STATUS_OK = "ok";
-
-  /**
-   * The Constant STATUS_SKIPPED.
-   */
-  public static final String STATUS_SKIPPED = "skipped";
-
-  /**
-   * The assertions.
-   */
-  @XStreamAsAttribute
-  private int assertions;
+  public enum Status {
+    OK, SKIPPED, FAILURE, ERROR
+  }
 
-  /**
-   * The class name.
-   */
   @XStreamAsAttribute
   @XStreamAlias("class")
   private String className;
```

Comment:
@inverno can't we remove unused fields? `className`, `name`, `tests`, `time`, `assertions`